core/governance/market.go

```diff
@@ -127,6 +142,24 @@ func assignRiskModel(definition *types.NewMarketConfiguration, target *types.Tra
 	return nil
 }
 
+func assignSpotRiskModel(definition *types.NewSpotMarketConfiguration, target *types.TradableInstrument) error {
+	switch parameters := definition.RiskParameters.(type) {
+	case *types.NewSpotMarketConfigurationSimple:
+		target.RiskModel = &types.TradableInstrumentSimpleRiskModel{
+			SimpleRiskModel: &types.SimpleRiskModel{
```

Comment:
Seeing as spot markets are new, I'm wondering whether we should add the simple risk model here. It's a bit of a relic of the early days where we had to test margin calculation stuff, and used a simple risk model to quickly set the margin levels/factors (maintenance, search, and release). As I understand it, it's not something we intend to be used in the wild, maybe we should only support the _actual_ risk models (lognormal) for new features like this

Reply:
ideally simple should only be for internal use because it makes things easy for feature testing but for now I wouldn't worry about it. I'm not sure tbh how much of a risk model would be left for spots when the probability of trading things goes away with the SLA liquidity